dev/ese/published/inc/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+configure_file(bf/bfftl.hxx ${INC_OUTPUT_DIRECTORY}/bfftl.hxx COPYONLY)
+configure_file(bf/_bfconst.hxx ${INC_OUTPUT_DIRECTORY}/_bfconst.hxx COPYONLY)
+configure_file(cc.hxx ${INC_OUTPUT_DIRECTORY}/cc.hxx COPYONLY)
+configure_file(collection.hxx ${INC_OUTPUT_DIRECTORY}/collection.hxx COPYONLY)
+configure_file(dht.hxx ${INC_OUTPUT_DIRECTORY}/dht.hxx COPYONLY)
+configure_file(errdata.hxx ${INC_OUTPUT_DIRECTORY}/errdata.hxx COPYONLY)
+configure_file(eseshadow.h ${INC_OUTPUT_DIRECTORY}/eseshadow.h COPYONLY)
+configure_file(iterquery.hxx ${INC_OUTPUT_DIRECTORY}/iterquery.hxx COPYONLY)
+configure_file(jet.h ${INC_OUTPUT_DIRECTORY}/jet.h COPYONLY)
+configure_file(jethdr.w ${INC_OUTPUT_DIRECTORY}/eseex_x.h COPYONLY)
+configure_file(jethdr.w ${INC_OUTPUT_DIRECTORY}/jet/eseex_x.h COPYONLY)
```

Comment:
This is weird, we should need a copy in inc output AND in inc output\jet\?

Reply:
To be honest, I don't completely remember why I had to do this - this file came from my "other OS" branch, and I do use that path in the OS lib. I'll look later if I can remove it (don't want to touch it in this PR)